virttest/migration.py

```diff
@@ -700,11 +700,12 @@ def check_result(self, result, params):
 
         if status_error:  # Migration should fail
             if err_msg:   # Special error messages are expected
-                if not re.search(err_msg, result.stderr_text.strip()):
-                    raise exceptions.TestFail("Can not find the expected "
-                                              "patterns '%s' in output '%s'"
-                                              % (err_msg,
-                                                 result.stderr_text.strip()))
+                err_msg_list = err_msg.split('|')
+                if not any(re.search(patt.strip(), result.stderr_text.strip())
```

Comment:
there is one method named: check_status_output in virttest/utils_test/libvirt.py, not sure whether it can satisfy your request.